packages/create-cloudflare/src/git.ts

```diff
@@ -83,10 +58,15 @@ export const offerGit = async (ctx: C3Context) => {
 };
 
 export const gitCommit = async (ctx: C3Context) => {
+	assert.notStrictEqual(
+		ctx.args.git,
+		undefined,
+		"Expected git context to be defined by now",
+	);
 	// Note: createCommitMessage stores the message in ctx so that it can
 	//       be used later even if we're not in a git repository, that's why
 	//       we unconditionally run this command here
-	const commitMessage = await createCommitMessage(ctx);
+	ctx.commitMessage = await createCommitMessage(ctx);
```

Comment:
Comment:

That's indeed confusing - maybe renaming the property to "creationMessage" and using that for the commit message could help alleviate some confusion?

(not asking to chnage in this PR)